### PR TITLE
[server] Possibility to set WMS featureInfo tolerance

### LINKED ISSUE #580: [server] Possibility to set WMS featureInfo tolerance in request. Funded by city of Uster
possibility to set WMS featureInfo tolerance in request. Funded by city of Uster

https://github.com/qgis/QGIS/commit/2d0f08bd59eabed08bf7bcaf9834d9ae494514a3

### PR DESCRIPTION
**Pull request by yjacolin**
* cleanup
* change QGIS version number tothe correct version

Fix #580

### CODE REVIEW

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 1]
```diff
@@ -439,29 +439,29 @@ OCG WMS 1.3.0 specification:
...
 
-* **FI_POINT_TOLERANCE** parameter: (Available in QGIS 2.10 and above)
+* **FI_POINT_TOLERANCE** parameter: (Available in QGIS 2.12 and above)
     Tolerance for point layers *GetFeatureInfo* request, in pixels.
-* **FI_LINE_TOLERANCE** parameter: (Available in QGIS 2.10 and above)
+* **FI_LINE_TOLERANCE** parameter: (Available in QGIS 2.12 and above)
     Tolerance for linestring layers *GetFeatureInfo* request, in pixels.
-* **FI_POLYGON_TOLERANCE** parameter: (Available in QGIS 2.10 and above)
+* **FI_POLYGON_TOLERANCE** parameter: (Available in QGIS 2.12 and above)
     Tolerance for polygon layers *GetFeatureInfo* request, in pixels.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: Do we need to keep the release of availability, given that user is supposed (and invited) to have a 2.14 version?

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 2]
```diff
@@ -482,26 +482,26 @@ Parameters in the GetPrint request are:
...
-* <map_id>:SCALE Sets a mapscale to a composer map. This is useful to ensure 
-  scale based visibility of layers and labels even if client and server may 
+* <map_id>:SCALE Sets a mapscale to a composer map. This is useful to ensure
+  scale based visibility of layers and labels even if client and server may
   have different algorithms to calculate the scale denominator
-* <map_id>:LAYERS, <map_id>:STYLES possibility to give layer and styles list 
-  for composer map (useful in case of overview maps which should have only a 
+* <map_id>:LAYERS, <map_id>:STYLES possibility to give layer and styles list
+  for composer map (useful in case of overview maps which should have only a
   subset of layers)
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: just wonder if it makes sense to have these parameters also in bold, as done for the others

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 3]
```diff
@@ -547,9 +547,9 @@ not available in the GetCapabilities output:
 DXF Export
 ------------
 
-Starting with QGIS 2.11 (QGIS master as of 2015-07) it is now possible to 
-export layers in the DXF format using the GetMap Request. Only layers that 
-have read access in the WFS service are exported in the DXF format. Here is a 
+Starting with QGIS 2.11 (QGIS master as of 2015-07) it is now possible to
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: same comment as above about date, simply "It is possible to ..."

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 4]
```diff
@@ -562,35 +562,35 @@ Parameters:
...
 
-* SCALE:scale to be used for symbology rules, filters and styles (not an 
+* SCALE:scale to be used for symbology rules, filters and styles (not an
   actual scaling of the data - data remains in the original scale)
-* MODE:NOSYMBOLOGY|FEATURESYMBOLOGY|SYMBOLLAYERSYMBOLOGY corresponds to the 
+* MODE:NOSYMBOLOGY|FEATURESYMBOLOGY|SYMBOLLAYERSYMBOLOGY corresponds to the
   three export options offered in the QGIS Desktop DXF export dialogue
-* LAYERSATTRIBUTES:yourcolumn_with_values_to_be_used_for_dxf_layernames - if 
+* LAYERSATTRIBUTES:yourcolumn_with_values_to_be_used_for_dxf_layernames - if
   not specified, the original QGIS layer names are used
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: same comment about bolding parameters

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst (hunk 3 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: Actually, this is already fixed in #1062

**Comment by yjacolin** [comment]
Ready to merge as soon test passes.

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 5]
```diff
@@ -480,28 +480,28 @@ The client has now the information to request a print output::
...
+* **<map_id>**:GRID_INTERVAL_X, <map_id>:GRID_INTERVAL_Y Grid line density for a
   composer map in x- and y-direction
-* <map_id>:SCALE Sets a mapscale to a composer map. This is useful to ensure 
-  scale based visibility of layers and labels even if client and server may 
+* **<map_id>**:SCALE Sets a mapscale to a composer map. This is useful to ensure
+  scale based visibility of layers and labels even if client and server may
   have different algorithms to calculate the scale denominator
-* <map_id>:LAYERS, <map_id>:STYLES possibility to give layer and styles list 
-  for composer map (useful in case of overview maps which should have only a 
+* **<map_id>:LAYERS, <map_id>**:STYLES possibility to give layer and styles list
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: @yjacolin I was thinking about something like **\<map_id>:EXTENT** as done in https://github.com/qgis/QGIS-Documentation/pull/1059/files#diff-1fd0a6614c78784a60c8b130f555fdddR565

Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: my bad, it was I wanted to do! Thanks to note this.

**Pull request merged**